app/admin/api/v3/profile.rb

```diff
@@ -19,6 +21,33 @@ def clear_cache
       input :name, as: :select,
                    collection: Api::V3::Profile::NAME,
                    hint: object.class.column_comment('name')
+
+
+      input :main_topojson_path,
+            hint: 'Path must be relative to https://github.com/Vizzuality/trase/tree/develop/frontend/public/vector_layers and start with / ',
```

Comment:
The hints in admin form would be better defined in the `schema_comments.yml` file, so that they can show up in the schema documentation.